app/angular/tsconfig.json

```diff
@@ -8,6 +8,7 @@
       "node"
     ],
     "rootDir": "./src",
+    "skipLibCheck": true,
```

Comment:
I tried, but app/angular just seems to have many issues I personally do not know how to solve. I solved some of them... But not all. Skipping for now.